frontend-web/webclient/app/Project/Grant/GrantApplicationEditor.tsx

```diff
@@ -1134,9 +1243,9 @@ const PostCommentWrapper = styled.form`
 `;
 
 const HelpText = styled.p`
-    margin: 0;
-    font-size: ${theme.fontSizes[1]}px;
-    color: var(--gray, #f00);
+  margin: 0;
+  font-size: ${theme.fontSizes[1]}px;
+  color: var(--gray, #ff0000);
```

Comment:
Was this not working? The #f00 (equivalent to #ff000) is intended to only show a garish red as a fallback when gray isn't being shown correctly.